modules/vaos/app/services/vaos/v2/eps/redis_client.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'common/models/redis_store'
+require 'common/models/concerns/cache_aside'
+
+module VAOS
+  module V2
+    module Eps
+      class RedisClient < Common::RedisStore
+        include Common::CacheAside
+
+        redis_config_key :eps_appointment_access_token
+
+        def store(access_token)
```

Comment:
wouldn't we need to define an attribute instead?

Reply:
Addressed in [1fd553f7](https://github.com/department-of-veterans-affairs/vets-api/pull/19307/commits/1fd553f7a06ace58dd8c9ca47ffd7a5023292968)